src/theory/arith/theory_arith_private.h

```diff
@@ -713,8 +713,8 @@ class TheoryArithPrivate {
     return (d_containing.d_valuation).getSatValue(n);
   }
 
-  context::CDQueue<Node> d_approxCuts;
-  std::vector<Node> d_acTmp;
+  context::CDQueue<TrustNode> d_approxCuts;
```

Comment:
Can you add some quick doc on these members?

Reply:
I have rather limited understanding of how these are used, but I documented the (very little) that I know in aa52080e0

Reply:
No worries :)